salt/utils/templates.py

```diff
@@ -345,11 +346,17 @@ def render_jinja_tmpl(tmplstr, context, tmplpath=None):
                                        **env_args)
 
     jinja_env.filters['strftime'] = salt.utils.date_format
-    jinja_env.filters['sequence'] = salt.utils.jinja.ensure_sequence_filter
     jinja_env.filters['yaml_dquote'] = salt.utils.yamlencoding.yaml_dquote
     jinja_env.filters['yaml_squote'] = salt.utils.yamlencoding.yaml_squote
     jinja_env.filters['yaml_encode'] = salt.utils.yamlencoding.yaml_encode
 
+    for function_name in context['salt'].keys():
+        # Simply iterating over the keys forces the execution modules to load
```

Comment:
> Simply iterating over the keys forces the execution modules to load

I don't think so - when the template is rendered, the ```__salt__``` object is injected under the key ```salt``` of the context (e.g.: https://github.com/saltstack/salt/blob/7dd6d2a47fb755178c80a65a193d55517c78130e/salt/modules/file.py#L3614).
So the for loop is a simple iteration over the keys of a dictionary, whose keys are function objects.